pipelines/managed/e2e/e2e.yaml

```diff
@@ -4,7 +4,7 @@ kind: Pipeline
 metadata:
   name: e2e
   labels:
-    app.kubernetes.io/version: "3.1.0"
+    app.kubernetes.io/version: "3.1.1"
```

Comment:
If I am not mistaken when adding/removing parameters, the minor part in the version should be bumped, not the patch.

Reply:
updated all pipeline yamls and READMEs